pkg/kn/commands/plugin/handler.go

```diff
@@ -76,6 +76,17 @@ func (h *DefaultPluginHandler) Lookup(name string) (string, bool) {
 			return pluginDirPluginPath, true
 		}
 
+		// Try to match well-known file extensions on Windows
```

Comment:
@dsimansk while we are at it, could you please also adapt the three lines above (just to be consistent) and add a comment that we should strive for a more sophisticated error handling in the future ?

Reply:
Sure, will do.